src/main/markdown/doc/latest/DevGuideI18nLocale.md

```diff
@@ -76,9 +76,13 @@ are probably better off choosing some real language as a default, such as
 the following in your [module XML](DevGuideOrganizingProjects.html#DevGuideModuleXml):
 
 ```
-<set-property-fallback name="locale" value="en"/>
+<extend-property name=locale" value"es,de,fr,en,it" <!-- 1. full list of supported locales --> 
+<set-property-fallback name="locale" value="en"/> <!-- 2. fallback locale -->
+<set-property name="locale" value="es,de,fr,en,it"/> <!-- 3. full list of supported localesm again -->
```

Comment:
How about:
```xml
<extend-property name="locale" values="en,es,de,fr,it" /> <!-- 1. full list of supported locales --> 
<set-property name="locale" value="en,es,de,fr,it" />     <!-- 2. remove the 'default' locale permutation -->
<set-property-fallback name="locale" value="en" />        <!-- 3. set the fallback locale, in replacement for the default 'default' locale -->
```

Maybe move the comments before each line, depending on the rendering (to avoid any horizontal scrolling)

Reply:
you know, at some point it would be faster if you just write an acceptable version yourself ;)

Reply:
This thought did actually cross my mind :slightly_smiling_face: 

I'd love another reviewer to chime in though, and a discussion is better than counter-proposal IMO.